shared/samplecomposemultiplatform/build.gradle.kts

```diff
@@ -30,3 +35,45 @@ dependencies {
 
     ktlintRuleset(libs.ktlint.composeRules)
 }
+
```

Comment:
Thinking about it, could the whole CSB configuration be extracted to its own convention plugin?

Reply:
Yeah, I wanted to try it as well, since it's a lot of ugly code 😄

Reply:
It seems it's not possible or at least I can't find a way how to do it, too many unresoled things visible only in build.gradle after adding the plugins and syncing 😞